pkg/chaosdaemon/httpchaos_server.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package chaosdaemon
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/shirou/gopsutil/process"
+
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+	"github.com/chaos-mesh/chaos-mesh/pkg/bpm"
+	pb "github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb"
+)
+
+const (
+	tproxyBin    = "/usr/local/bin/tproxy"
+	rustLog      = "RUST_LOG"
+	rustLogLevel = "trace"
+	pathEnv      = "PATH"
+)
+
+func (s *DaemonServer) ApplyHttpChaos(ctx context.Context, in *pb.ApplyHttpChaosRequest) (*pb.ApplyHttpChaosResponse, error) {
+	log.Info("applying http chaos", "Request", in)
```

Comment:
It seems we do not implement the recover things.

Reply:
Just update resource with empty proxy ports to recover.

Reply:
OK, but it sounds not so good. 🤪